docs/developer/workflows.md

```diff
@@ -66,3 +66,15 @@ These workflows will help with excluding certain terms from integration into Mon
 7. `reports/excluded_terms.txt`: Runs reports/%_term_exclusions.txt for all ontologies and combines into a single file.
 8. `reports/exclusion_reasons.robot.template.tsv`: Runs reports/%_exclusion_reasons.robot.template.tsv for all ontologies and combines into a single file. 
 9. `exclusions-all`: Runs all exclusion artefacts for all ontologies.
+
+## Synchronization
+These workflows help synchronize Mondo with source ontologies.
+
+#### Makefile goals
+1. `generate-synchronization-files`: Runs synchronization pipeline.
+2. `sync-subclassof`: Runs 'sync-subclassof' part of synchronization pipeline, generating set of outputs for all ontologies.
+3. `sync-subclassof-%`: Generates subClassOf synchronization outputs for given ontology. Alias for `reports/%.subclass.added.robot.tsv`, `reports/%.subclass.confirmed.robot.tsv`, and `reports/%.subclass.in-mondo-only.tsv`.
+4. `reports/%.subclass.added.robot.tsv`: Creates robot template containing new subclass relationships from given ontology to be imported into Mondo. Running this also runs / generates `reports/%.subclass.confirmed.robot.tsv` and `reports/%.subclass.in-mondo-only.tsv`.
+5. `reports/%.subclass.confirmed.robot.tsv`: Creates robot template containing subclass relations for given ontology that exist in Mondo and are confirmed to also exist in the source. Running this also runs / generates `reports/%.subclass.added.robot.tsv` and `reports/%.subclass.in-mondo-only.tsv`.
+6. `reports/%.subclass.in-mondo-only.tsv`: Path to create file for relations for given ontology where direct subclass relation exists only in Mondo and not in the source. Running this also runs / generates `reports/%.subclass.added.robot.tsv` and `reports/%.subclass.confirmed.robot.tsv`.
```

Comment:
We need these (reports/%.subclass.in-mondo-only.tsv) analytically (for the paper), but do not check them into the repo (.gitignore). Also, they seem quite big - make sure that both subject and object are actually in the source.

Reply:
- [ ] 1. `.gitignore`: I hear you on that! I have the goal delete them after aggregating into `reports/sync-subClassOf.in-mondo-only.tsv`, but I should/will add to `.gitignore`.